ansible/install.yml

```
@@ -18,6 +18,15 @@
       include_role:
         name: geerlingguy.docker
 
+    - name: Download minio
+      get_url:
+        url: https://dl.min.io/server/minio/release/linux-amd64/minio
+        dest: /usr/local/bin/minio
+        mode: 'a+x'
+
+    - name: Install systemd unit file minio
+      template: src=minio.service.j2 dest=/etc/systemd/system/minio.service
+
     - name: Install systemd unit file
```

```suggestion
    - name: Install systemd unit files hashistack
```
Might as well change this while we're at it.